### PR TITLE
chore(bench): benches optimization

### PR DESCRIPTION
**Pull request by lemarier**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [x] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [x] No

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
-/target
-Cargo.lock
+target
+/Cargo.lock
```
Comment (wusyong) [inline .gitignore]: Does it really have to add other lock files?

Comment (lemarier) [reply .gitignore]: its used by the cache it would work without tho

Comment (lemarier) [reply .gitignore]: fixed

File: bench/src/utils.rs [hunk 2]
```diff
@@ -2,23 +2,68 @@
...
+  pub errors: u64,
+}
+
+pub fn get_target() -> &'static str {
+  #[cfg(target_os = "macos")]
+  return "x86_64-apple-darwin";
+  #[cfg(target_os = "linux")]
+  return "x86_64-unknown-linux-gnu";
+  #[cfg(target_os = "windows")]
+  return unimplemented!();
```
Comment (wusyong) [inline bench/src/utils.rs:45]: Is this intended?

Comment (lemarier) [reply bench/src/utils.rs:45]: yup gonna work later to bring benches to other platforms but limited as the CI require X server.

**Review by wusyong — APPROVED** [review]

**Pull request merged**